sdk/python/kfp/cli/component_test.py

```diff
@@ -576,41 +576,42 @@ def test_existing_dockerfile_can_be_overwritten(self):
                 COPY . .
                 '''))
 
-    def test_dockerfile_can_contain_custom_kfp_package(self):
-        component = _make_component(
-            func_name='train', target_image='custom-image')
-        _write_components('components.py', component)
-        package_dir = os.path.dirname(os.path.dirname(self.current_dir))
-
-        # suppresses large stdout from subprocess that builds kfp package
-        with mock.patch.object(
-                subprocess,
-                'run',
-                new=functools.partial(subprocess.run, capture_output=True)):
-            result = self.runner.invoke(
-                self.cli,
-                [
-                    'build',
-                    str(self._working_dir), f'--kfp-package-path={package_dir}'
-                ],
-            )
-        self.assertEqual(result.exit_code, 0)
-        self._docker_client.api.build.assert_called_once()
-        self.assert_file_exists('Dockerfile')
-        with open('Dockerfile', 'r') as f:
-            contents = f.read()
-        file_start = textwrap.dedent('''\
-                # Generated by KFP.
-
-                FROM python:3.8
-
-                WORKDIR /usr/local/src/kfp/components
-                COPY runtime-requirements.txt runtime-requirements.txt
-                RUN pip install --no-cache-dir -r runtime-requirements.txt
-                ''')
-        self.assertTrue(contents.startswith(file_start))
-        self.assertRegex(contents, 'RUN pip install --no-cache-dir kfp-*')
 
+# Commenting this test out as it's failing.
```

Comment:
Instead of commenting out the test, can we use `@unittest.skip`? This approach preserves the code, helping us avoid breaking it and maintaining easy access to references.

More info about `@unittest.skip`: https://docs.python.org/3/library/unittest.html#skipping-tests-and-expected-failures

Reply:
Can you also add a comment with a link to https://github.com/kubeflow/pipelines/issues/11038?

Reply:
Done, added `@unittest.skip` statement instead of commenting out the test, and added a comment with a link to https://github.com/kubeflow/pipelines/issues/11038